inference-engine/tests/ie_test_utils/functional_test_utils/layer_test_utils.cpp

```diff
@@ -80,10 +80,10 @@ void LayerTestsCommon::ConfigureNetwork() const {
     }
 }
 
-void LayerTestsCommon::LoadNetwork() {
+void LayerTestsCommon::LoadNetwork(const std::map<std::string, std::string>& config) {
     cnnNetwork = InferenceEngine::CNNNetwork{function};
     ConfigureNetwork();
-    executableNetwork = core->LoadNetwork(cnnNetwork, targetDevice);
+    executableNetwork = core->LoadNetwork(cnnNetwork, targetDevice, config);
```

Comment:
IMO, Config should be a field of class. As a result, we should not give a config to the function.

Reply:
Should I create new `LayerTestsCommon` type member specially to configure device during `LoadNetwork`?   
Note, please, there is `std::map<std::string, std::string> configuration;` member of `LayerTestsCommon` type which is use as `core->SetConfig(configuration, targetDevice);`  already.